scvelo/plotting/velocity_embedding_stream.py

```diff
@@ -260,6 +260,6 @@ def velocity_embedding_stream(
             **scatter_kwargs,
         )
 
-        savefig_or_show(dpi=dpi, save=save, show=show)
+        savefig_or_show("embedding_stream", dpi=dpi, save=save, show=show)
```

Comment:
This will lead to unexpected and undesired behavior if `save` is passed as a string (see [here](https://github.com/theislab/scvelo/blob/cecea01849a7309358b22235c7b333e99af0838c/scvelo/plotting/utils.py#L954-L969)). Would you please add an if clause prior to the function call and set `save` to `"embedding_key"` if `save` is a boolean and `True`?

Reply:
Done!